third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -1266,7 +1288,7 @@ def align(
                 Align on index (0), columns (1), or both (None).
 
         Returns:
-            tuple of (DataFrame, type of other): Aligned objects.
+            tuple of (bigframes.pandas.DataFrame, type of other): Aligned objects.
```

Comment:
Let's use Python type syntax here.

```suggestion
            Tuple[bigframes.pandas.DataFrame, type of other]: Aligned objects.
```